src/context.rs

```diff
@@ -277,19 +279,23 @@ impl Context {
         )
     }
 
+    /// Declares a unit value of the given sort
```

Comment:
It isn't a unit value but an unconstrained value, right?

And it seems fine to me to have sugar/helpers for common operations, even if you can technically implement the operation in terms of others.

Reply:
Updated comments!